UDV-Core/src/Modules/Temporal/TemporalModule.js

```diff
@@ -71,7 +73,22 @@ export class TemporalModule {
 
         // Instantiate the temporal window
         // TODO: make it active by default
-        this.temporalWindow = new TemporalWindow(refreshCallback, temporalOptions);
+
+        // load options from baseconfig
+        let n = new NetworkManagerSingleton();
+        n.list_option = temporalOptions.graphOption;
+
+
+        // Choose the window type you want
+        // For time slider :
```

Comment:
instead of commenting/uncommenting lines, i'd rather like that we can pass this information in the options of the constructor with something like:
````
{
        minTime: XXXX,
        maxTime: YYYY,
        currentTime: ZZZZ,
        timeStep: A,
        temporalwindow: {
           name: <graphWindow or sliderWindow> //please use an enum for that 
           options: {
              // empty if sliderWindow and contains graph options if graphWindow
           }
        }
        graphOption: baseDemo.config['graphOption'] // the window is encapsulate by TemporalModule so we need to pass by it for the graphic options
````

Reply:
The choice is now made in the generalconfi file.